modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -458,7 +478,7 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
 
     const keychainArray: Triple<Keychain> = [keychains.user, keychains.backup, keychains.bitgo];
 
-    const keySignatures = _.get(wallet, '_wallet.keySignatures');
+    const keySignatures = _.get(wallet, '_wallet.keySignatures', {});
```

Comment:
nit: we probably should start getting rid of lodash overtime, but doesn't need to be in this PR